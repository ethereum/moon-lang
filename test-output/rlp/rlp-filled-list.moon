either = zb2rhjDvcJ6d6xsRH5o6ZAsVmM7CGhRTZo2ADCQKtuXE4ijzy
encode = zb2rhhoXXkduCezoqZtAiNQ4Mggz1KDZBtyoagoEbRcjJEX5H
nil    = zb2rhbQFhrUA2SqQVKweLeRoHomXzJwAsKLtKT2TjW7Eunk6x
cons   = zb2rhcXRC5mtMz8yYiWuZourtH3MaZwx4m3VfDRPvvFkwpbFh
List   = (either "Left")
String = (either "Right")
filled = (List (cons (String "cat") (cons (String "dog") nil)))
(encode filled)

