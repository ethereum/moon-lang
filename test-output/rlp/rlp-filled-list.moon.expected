"0xc88363617483646f67"
